apps/prairielearn/elements/pl-code/pl-code.css

```diff
@@ -57,3 +57,11 @@
 .pl-code > div {
   overflow: auto;
 }
+
+.pl-code .prevent-select {
```

Comment:
nit: change `prevent-select` here to `pl-code-prevent-select`. 

It is nice to have this sort of 'namespaceing' on element-specific classes that aren't used elsewhere.

Reply:
[7a88aae](https://github.com/PrairieLearn/PrairieLearn/pull/11939/commits/7a88aae8d9cb18d463d929f3c3a9f6e3c2fe270d)